src/components/about-us/opening/background.js

```diff
@@ -0,0 +1,179 @@
+import { screen } from '../utils/screen'
+import { intro } from '../constants/data/introduction'
+import billboard from '../../../../static/asset/about-us/billboard.png'
+import React, { PureComponent } from 'react'
+import styled from 'styled-components'
+
+const TriangleSizes = {
+  desktop: {
+    width: '653px',
+    height: '112px'
+  },
+  mobile: {
+    width: '43px',
+    height: '276px'
+  },
+  tablet: {
+    width: '102px',
+    height: '661px'
+  }
+}
+
+const DesktopTriangles = styled.div`
+  position: absolute;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  ${screen.tabletBelow`
+    display: none;
+  `}  
+`
+
+const MobileTriangles = styled.div`
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  right: 0;
+  ${screen.desktopAbove`
+    display: none;
+  `}
+`
+const Triangle = styled.div`
+  width: ${TriangleSizes.desktop.width};
+  height: ${TriangleSizes.desktop.height};
+  border-top: solid ${TriangleSizes.desktop.height} #c7000a;
+  border-left: solid calc(${TriangleSizes.desktop.width}*3/8) transparent;
+  border-right: solid calc(${TriangleSizes.desktop.width}*5/8) transparent;
+  &:before,&:after{
+    box-sizing: border-box;
+  }  
+`
+
+const TriangleRightTop = Triangle.extend`
+  position: absolute;
+  right: 0;
+  top: 0;
+`
+
+const TriangleRightBottom = Triangle.extend`
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  transform: rotateX(180deg);  
+`
+
+const MobileTriangle = styled.div`
+  width: ${TriangleSizes.mobile.width};
+  height: ${TriangleSizes.mobile.height};
+  border-left: solid ${TriangleSizes.mobile.width} #c7000a;
+  border-top: solid calc(${TriangleSizes.mobile.height}*3/8) transparent;
+  border-bottom: solid calc(${TriangleSizes.mobile.height}*5/8) transparent;
+  &:before,&:after{
+    box-sizing: border-box;
+  }
```

Comment:
`box-sizing: border-box` seems already to be defined in the `src/helpers/Html.js`